### DISCUSSION TITLE
Can't see my 3rd fan for my evga 3080 ti

### DISCUSSION THREAD

**Discussion by DrMental** [Rem0o/FanControl.Releases#1966] [Q&A]
![image](https://github.com/Rem0o/FanControl.Releases/assets/140925436/4d2fee75-ea76-409a-b69c-457b77d25f86)

![image](https://github.com/Rem0o/FanControl.Releases/assets/140925436/9112a34c-2ff4-4510-ba22-d4dd6d3ad006)

**Comment by Rem0o** [comment]
NvAPI only exposes 2 channels.  Control 1 should control 2 of those 3, while the other channel should control the 3rd fan.
